backend/src/services/sessionService.js

```diff
@@ -25,15 +25,12 @@ export const sessionService = {
 const checkUser = async (username, password) => {
   const user = await getUser(username, password);
```

Comment:
@kkeszthe instead of using the repo, use the userService for this and that should return a single User object, same with the kindgom